client/config.go

```diff
@@ -0,0 +1,181 @@
+package client
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/mitchellh/go-homedir"
+	"bufio"
+	"path"
+	"os"
+	"github.com/pkg/errors"
+	"encoding/json"
+	"io/ioutil"
+	"github.com/pelletier/go-toml"
+	"fmt"
+)
+
+type cliConfig struct {
+	Home      string `toml:"home"`
+	ChainID   string `toml:"chain_id"`
+	TrustNode bool   `toml:"trust_node"`
+	Encoding  string `toml:"encoding"`
```

Comment:
Can there be a comment what the encoding flag is for? I'm confused why its "btc" / what it does.

Reply:
It's the same encoding flag as in Tendermint - i.e. whether the encoding is bech32, json, etc.

Reply:
But what does `encoding := "btc"` mean, and what is it the encoding for? Pubkeys / addresses should always be bech32, not sure what JSON encoding of a pubkey means.

Reply:
Sorry, JSON is a misnomer. It's this flag:

```
  -e, --encoding string   Binary encoding (hex|b64|btc) (default "hex")
```